specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-01-30-preview/definitions/Tasks.json

```diff
@@ -0,0 +1,1017 @@
+        },
+        "output": {
+          "description": "Task output. This is ignored if submitted.",
+          "type": "array",
+          "items": {
+            "$ref": "./ConnectToSourcePostgreSqlSyncTask.json#/definitions/ConnectToSourcePostgreSqlSyncTaskOutput"
+          },
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProjectTaskProperties"
+        }
+      ]
+    },
+    "ConnectToSourceMySqlTaskProperties": {
+      "x-ms-discriminator-value": "ConnectToSource.MySql",
+      "type": "object",
+      "description": "Properties for the task that validates MySQL database connection",
+      "properties": {
+        "input": {
+          "description": "Task input",
+          "$ref": "./ConnectToSourceMySqlTask.json#/definitions/ConnectToSourceMySqlTaskInput"
+        },
+        "output": {
+          "description": "Task output. This is ignored if submitted.",
+          "type": "array",
+          "items": {
+            "$ref": "./TasksCommon.json#/definitions/ConnectToSourceNonSqlTaskOutput"
+          },
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./Tasks.json#/definitions/ProjectTaskProperties"
+        }
+      ]
+    },
+    "ConnectToSourceOracleSyncTaskProperties": {
+      "x-ms-discriminator-value": "ConnectToSource.Oracle.Sync",
+      "type": "object",
+      "description": "Properties for the task that validates Oracle database connection",
+      "properties": {
+        "input": {
+          "description": "Task input",
+          "$ref": "./OracleAzureDbPostgreSqlSyncTask.json#/definitions/ConnectToSourceOracleSyncTaskInput"
+        },
+        "output": {
+          "description": "Task output. This is ignored if submitted.",
+          "type": "array",
+          "items": {
+            "$ref": "./OracleAzureDbPostgreSqlSyncTask.json#/definitions/ConnectToSourceOracleSyncTaskOutput"
+          },
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProjectTaskProperties"
+        }
+      ]
+    },
+    "ConnectToTargetSqlDbTaskProperties": {
+      "x-ms-discriminator-value": "ConnectToTarget.SqlDb",
+      "type": "object",
+      "description": "Properties for the task that validates connection to SQL DB and target server requirements",
+      "properties": {
+        "input": {
+          "description": "Task input",
+          "$ref": "./ConnectToTargetSqlDbTask.json#/definitions/ConnectToTargetSqlDbTaskInput"
+        },
+        "output": {
+          "description": "Task output. This is ignored if submitted.",
+          "type": "array",
+          "items": {
+            "$ref": "./ConnectToTargetSqlDbTask.json#/definitions/ConnectToTargetSqlDbTaskOutput"
+          },
+          "readOnly": true
+        },
+        "createdOn": {
```

Comment:
>createdOn [](http://example.com/codeflow?start=9&length=9)

this should be coming from the systemData common contract on this resource.  If this is an s360 fix since you already have this property its fine.  If its a new property please do not add it

Reply:
applies to the other 2 places this property was added

Reply:
This was added because this was an s360 error .  I have now added systemData instead. System data has "createdAt" instead of "createdOn" , will this still resolve the s360 error?

Reply:
This is not where systemData lives in the resource.  The resources already have systemData at the correct place on line 132.  I can't help you decide what will/won't fix your S360 errors.  However, if this is a brand new property you aren't exposing today then do not add it.  If its something that is already exposed in your API versions (but not documented in swagger) then keep it

Reply:
Alright, "createdOn" was  already there and is not a brand new property so we can keep it